app/controllers/admin/tutorials_controller.rb

```diff
@@ -6,9 +6,9 @@ def edit
   def create 
     tutorial = Tutorial.create(new_tutorial_params)
     if tutorial.save
-      YoutubeResults.new.create_videos(tutorial)
-      flash[:notice] = "Successfully created tutorial. View it here."
+      YoutubeService.new.create_videos(tutorial)
       redirect_to admin_dashboard_path
+      flash[:notice] = "Successfully created tutorial. #{view_context.link_to("View it here.", tutorial_path(tutorial.id))}"
```

Comment:
view_context is a cool rails method I came across. This allows you to have a link within your flash message.

Reply:
Wait thats awesome. Im definitely going to use that in the future.